tensorflow/core/api_def/base_api/api_def_Add.pbtxt

```diff
@@ -2,6 +2,13 @@ op {
   graph_op_name: "Add"
   summary: "Returns x + y element-wise."
   description: <<END
+  Given two input tensors, tf.math.add computes and returns the element wise sum of the tensors.
+  Input range is `[-inf, inf]` and output range is `[-inf, inf]`.
```

Comment:
I don't think we should include input range if all inputs are supported. Similarly, we should not include the output range if any output can be provided.

Also, the intervals are wrong: from a mathematical point of view, the intervals should be open (`(-inf, inf)`) as the infinities cannot be reached. However, from a programming point of view not all values can be represented so maybe it's `[-float_max, float_max]` or something. Better to just not include the line at all.